.github/workflows/ci.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           - "0.17.0"
           - "0.18.0"
           - "0.19.0"
+          - "0.20.1"
```

Comment:
nit: `0.20.0` would be more in line with the others

Reply:
Hi, I understand but polars merge was added in 0.20 is okay to change to 0.20?

Reply:
0.20.0 in this contest means "<0.21.0", or "latest available before 0.21.0". That's how CI is set up. So yes, it's ok to do 0.20.0 here.